src/common/bit_util.h

```diff
@@ -41,6 +41,13 @@ inline size_t RawPopcount(const uint8_t *p, int64_t count) {
 
   return bits;
 }
+inline bool Int64OperationOverFlow(int64_t a, int64_t b, int64_t result, uint8_t operation) {
+  if (operation == 1) {
+    return __builtin_add_overflow(a, b, &result);
+  } else {
+    return __builtin_mul_overflow(a, b, &result);
+  }
+}
```

Comment:
This function looks very weird. What's `operation`? What's `1` here? Why `mul` is executed if `operation` is not 1?

Please remove it.

Reply:
Also, this function is totally wrong since the result is passed by value.